lib/counter_cache_rails.rb

```diff
@@ -1,3 +1,18 @@
 # frozen_string_literal: true
 
+require 'counter_cache_rails/configuration'
 require 'counter_cache_rails/railtie'
+
+module CounterCacheRails
+  class << self
+    def configuration
+      @configuration ||= Configuration.new(
+        expires_in: 1.hour
```

Comment:
If this is to be consistent with the way built-in counter_cache works, the cache should never expire. A COUNT(*) query is expensive, and doing it every hour would be bad for performance.

Reply:
I fixed expires_in option set nil by default https://github.com/iguchi1124/counter_cache-rails/pull/8/commits/30c844179ddf4afab8e196d456ed09cbaba54921